docs/record-encryption-quickstart/index.adoc

```diff
@@ -0,0 +1,254 @@
+----
+
+If you are using LocalStack, you need to create a key and an alias to that key.
+
+[source,shell]
+----
+KEY_ID=$(kubectl -n localstack exec deployments/localstack --  awslocal kms create-key --query KeyMetadata.KeyId --output text)
+kubectl -n localstack exec deployments/localstack -- awslocal kms create-alias --alias-name alias/KEK_trades --target-key-id ${KEY_ID}
+----
+
+== Produce some messages
+
+Now let's use Kafka's console producer CLI to send a few records to a topic called `trades`.
+The Record Encryption filter will encrypt the records before they reach broker, but this is completely transparent to the console producer.
+Ignore the warning about the UNKNOWN_TOPIC_OR_PARTITION, the topic will be created automatically.
+
+[source,shell]
+----
+echo 'IBM:100\nAPPLE:99' | kubectl run -n my-proxy -qi proxy-producer --image=quay.io/strimzi/kafka:0.47.0-kafka-4.0.0 --rm=true --restart=Never -- ./bin/kafka-console-producer.sh --bootstrap-server ${PROXIED_CLUSTER_BOOTSTRAP_SERVER} --topic trades
+----
+
+== Consume the messages
+
+Now let's use Kafka's console consumer to fetch the records.
+You'll see the two records we sent above.
+The Record Encryption filter will decrypt the records before they reach consumer, but this is completely transparent to the console consumer.
+
+[source,shell]
+----
+kubectl run -n my-proxy -qi proxy-consumer --image=quay.io/strimzi/kafka:0.47.0-kafka-4.0.0 --rm=true --restart=Never -- ./bin/kafka-console-consumer.sh  --bootstrap-server ${PROXIED_CLUSTER_BOOTSTRAP_SERVER} --topic trades --from-beginning --timeout-ms 10000
+----
+
+== Verify that the records are encrypted on the broker
+
+You want to see that the records are truly encrypted on the broker, right.  Let's use the console consumer again,
+but this time we'll consume straight from the Kafka cluster, bypassing the proxy.
+We'll get back the records, but they'll contain ciphertext, rather than the plaintext.
+
+[source,shell]
+----
+kubectl run -n kafka -qi cluster-consumer --image=quay.io/strimzi/kafka:0.47.0-kafka-4.0.0 --rm=true --restart=Never -- ./bin/kafka-console-consumer.sh  --bootstrap-server ${DIRECT_CLUSTER_BOOTSTRAP_SERVERS} --topic trades --from-beginning --timeout-ms 10000
+----
+
+= Cleaning up
+
+To clean up, remove the proxy and the kafka cluster
+
+[source,shell]
+----
+kubectl delete -f examples/record-encryption/
+kubectl delete -n kafka -f https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/refs/tags/0.47.0/examples/kafka/kafka-single-node.yaml
+----
+
+Remove the Kroxylicious Operator.
+
+[source,shell]
+----
+kubectl delete -f install
+----
+
+Use Helm to uninstall the remaining operators:
+
+[source,shell]
+----
+helm uninstall -n kafka strimzi-operator --ignore-not-found
+helm uninstall -n localstack localstack --ignore-not-found
+helm uninstall -n vault vault --ignore-not-found
+----
+
+= What next?
+
+Learn more about Record Encryption by referring to the the document.
```

Comment:
typo: [`the the`](https://open.spotify.com/track/3zf2eLN9Yf8ayZO5cn4Xh5?si=437f0997a2434892)